src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsServer.java

```diff
@@ -25,80 +25,83 @@
-     * The server must be bound using {@link #bind(InetSocketAddress,int)} before it can be used.
-     * The server must also have a HttpsConfigurator established with {@link #setHttpsConfigurator(HttpsConfigurator)}
-     * @throws IOException
+     * Creates a {@code HttpsServer} instance which is initially not bound to any
+     * local address/port. The {@code HttpsServer} is acquired from the currently
+     * installed {@link HttpServerProvider}. The server must be bound using
+     * {@link #bind(InetSocketAddress,int)} before it can be used. The server
+     * must also have a {@code HttpsConfigurator} established with
+     * {@link #setHttpsConfigurator(HttpsConfigurator)}.
+     *
+     * @throws IOException if an I/O error occurs
      */
-    public static HttpsServer create () throws IOException {
-        return create (null, 0);
+    public static HttpsServer create() throws IOException {
+        return create(null, 0);
     }
 
     /**
-     * Create a <code>HttpsServer</code> instance which will bind to the
-     * specified {@link java.net.InetSocketAddress} (IP address and port number)
+     * Create a {@code HttpsServer} instance which will bind to the specified
+     * {@link java.net.InetSocketAddress} (IP address and port number).
      *
      * A maximum backlog can also be specified. This is the maximum number of
-     * queued incoming connections to allow on the listening socket.
-     * Queued TCP connections exceeding this limit may be rejected by the TCP implementation.
-     * The HttpsServer is acquired from the currently installed {@link HttpServerProvider}
-     * The server must have a HttpsConfigurator established with {@link #setHttpsConfigurator(HttpsConfigurator)}
+     * queued incoming connections to allow on the listening socket. Queued TCP
+     * connections exceeding this limit may be rejected by the TCP implementation.
+     * The {@code HttpsServer} is acquired from the currently installed
+     * {@link HttpServerProvider}. The server must have a {@code HttpsConfigurator}
+     * established with {@link #setHttpsConfigurator(HttpsConfigurator)}.
      *
-     * @param addr the address to listen on, if <code>null</code> then bind() must be called
-     *  to set the address
-     * @param backlog the socket backlog. If this value is less than or equal to zero,
-     *          then a system default value is used.
+     * @param addr the address to listen on, if {@code null} then
+     *             {@link #bind(InetSocketAddress,int)}  must be called to set
+     *             the address
+     * @param backlog the socket backlog. If this value is less than or equal to
+     *               zero, then a system default value is used
      * @throws BindException if the server cannot bind to the requested address,
-     *          or if the server is already bound.
-     * @throws IOException
+     *          or if the server is already bound
+     * @throws IOException if an I/O error occurs
```

Comment:
This one needs to be listed in the CSR too.

Reply:
Change noted in CSR (https://bugs.openjdk.java.net/browse/JDK-8254968)